config/recipes/gclb/01-elastic-stack.yaml

```diff
@@ -0,0 +1,74 @@
+---
+apiVersion: elasticsearch.k8s.elastic.co/v1
+kind: Elasticsearch
+metadata:
+  name: hulk
+  labels:
+    app: hulk
+spec:
+  version: 7.6.1
+  http:
+    service:
+      metadata:
+        labels:
+          app: hulk
+        annotations:
+          # Enable TLS between GCLB and the application
+          cloud.google.com/app-protocols: '{"https":"HTTPS"}'
+          service.alpha.kubernetes.io/app-protocols: '{"https":"HTTPS"}'
+          # Uncomment the following line to enable container-native load balancing.
+          #cloud.google.com/neg: '{"ingress": true}'
```

Comment:
It seems like we probably want this to be the default, correct? If I'm reading the [docs](https://cloud.google.com/kubernetes-engine/docs/concepts/container-native-load-balancing) right there don't seem to be any substantial downsides

Reply:
If you forget to add the IP aliasing flag when creating the Kubernetes cluster, this will silently fail.  I assume that most people are still using the legacy cluster type so I kept this commented out for the time being.

Reply:
Ah the default in the GUI is to use vpc native clusters. I can see it either way. If you don't create it as VPC native in the CLI it yells at you at least:
WARNING: Currently VPC-native is not the default mode during cluster creation. In the future, this will become the default mode and can be disabled using --no-enable-ip-alias flag. Use --[no-]enable-ip-alias flag to suppress this warning.

But I can see how the silent failure is a pain to debug 🤷‍♀